### DISCUSSION TITLE
Visual Studio Extension "Signtool is not recognized"

### DISCUSSION THREAD

**Discussion by henrymcl** [wixtoolset/issues#6336] [Questions] [answered]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with? 

> **3.11.2** (I just downloaded right from https://wixtoolset.org/releases/)

* Which version of Visual Studio are you building with (if any)?

> **2019 Version 16.8.4**

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> **1.0.0.4**

* Which version of .NET are you building with?

> (.NET version)

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Windows 10

* Describe the problem and the steps to reproduce it / Describe the behavior you expected and how it differed from the actual behavior.

> Build error: 
```
3>		'signtool' is not recognized as an internal or external command,
3>		operable program or batch file.
3>		'signtool' is not recognized as an internal or external command,
3>		operable program or batch file.
3>		'signtool' is not recognized as an internal or external command,
3>		operable program or batch file.
3>		'signtool' is not recognized as an internal or external command,
3>		operable program or batch file.
```

**Comment by barnson** [comment, marked as answer]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions

**Reply by BMurri** [reply]
@henry132109, we need some detail as to what environment variables/project settings in the `wixproj` file you have when you run your build. At minimum, minimal repro steps.

**Reply by henrymcl** [reply]
The problem is resolved.
I was having issues with understanding the code for seb-win-factoring, but it's been resolved and the release builds are successfully signed.

Thanks all.
